Yoast/YoastSEO.js
# Add largest keyword distance assessment to assessor based on assessor options

## Summary

This PR can be summarized in the following changelog entry:

* n/a

## Relevant technical choices:

* Adds functionality that makes it possible to add the largest keyoword distance assessment based on assessor options.

## Test instructions

This PR can be tested by following these steps:

* Make a beta script combining this branch with `stories/1566-add-keywordDistribution-to-SEOAssessor-in-premium` in Yoast SEO Premium (see https://github.com/Yoast/wordpress-seo-premium/pull/1819).
* Add a long text (>200 words) and set a keyword. Add the keyword at the beginning and the end of the text. Make sure you get feedback about your keyword distribution.

Fixes #1566
